<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>198</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Дешифратор</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton:hover{color: blue; font: bold 10pt;}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="loadImgButton">
        <property name="text">
         <string>Загрузка изображения</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveImgButton">
        <property name="text">
         <string>Сохранение изображения</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="placeholderText">
       <string>Введите текст...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="outputLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Статус</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="decodeMsgButton">
        <property name="text">
         <string>Декодировка</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="encodeMsgButton">
        <property name="text">
         <string>Кодировка</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>loadImgButton</tabstop>
  <tabstop>saveImgButton</tabstop>
  <tabstop>encodeMsgButton</tabstop>
  <tabstop>decodeMsgButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
